store.py

```diff
@@ -122,17 +122,17 @@ def create_part(self, part):
     def update_part(self, part):
         """Update a part in the database, overwrite lcsc if supplied."""
         with contextlib.closing(sqlite3.connect(self.dbfile)) as con, con as cur:
-            if len(part) == 6:
-                cur.execute(
-                    "UPDATE part_info set value = ?, footprint = ?, lcsc = ?, exclude_from_bom = ?, exclude_from_pos = ? WHERE reference = ?",
-                    part[1:] + part[0:1],
-                )
-            else:
-                cur.execute(
-                    "UPDATE part_info set value = ?, footprint = ?, exclude_from_bom = ?, exclude_from_pos = ? WHERE reference = ?",
-                    part[1:] + part[0:1],
-                )
-
+            cur.execute(
```

Comment:
this is WAY simpler than the original version.

Reply:
Thats the point of this PR, I should have used a dict factory from the beginning, but wasn't aware that this is even possible ...